### PR TITLE
Correct README accessors desciprtion

### PR DESCRIPTION
**Pull request by pinealan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,8 +68,8 @@ yarn storybook
...
-| yAccessor         | ({ item }) => item                       | A function that takes each entry of `data` (named "item") as well as the index and returns the y-value of that entry                                                                                                                                           |
-| xAccessor         | ({ index }) => index                     | Same as `yAccessor` but returns the x-value of that entry                                                                                                                                                                                                      |
+| yAccessor         | ({ item }) => index                      | A function that takes each entry of `data` (named "item") as well as the index and returns the y-value of that entry                                                                                                                                           |
```
Comment (usrbowe) [inline README.md]: deconstructing `item`, but return `index`.

File: README.md [hunk 2]
```diff
@@ -68,8 +68,8 @@ yarn storybook
...
+| xAccessor         | ({ item }) => index                      | Same as `yAccessor` but returns the x-value of that entry                                                                                                                                                                                                      |
```
Comment (usrbowe) [inline README.md]: same here, but opposite.

**Comment by usrbowe** [comment]
@pinealan Can you provide more details on why to make proposed changes? Thanks.

**Comment by pinealan** [comment]
Sorry don't think this PR is actually needed. I was confused by the mappings in `chart.js` on [these lines](https://github.com/JesperLekland/react-native-svg-charts/blob/359c58b9fc0d25a6de5be9a4782fceafc1761324/src/chart/chart.js#L55-L61) when I made this PR.

I likely was confused since the variable name `item` were reused in different contexts. Would recommend renaming this part
```javascript
const mappedData = data.map((item, index) => ({
    y: yAccessor({ item, index }),
    x: xAccessor({ item, index }),
}))
const yValues = mappedData.map((point) => point.y) // item -> point
const xValues = mappedData.map((point) => point.x)
```

**Pull request closed without merging**